crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -331,16 +334,26 @@ export.get_block_hash
     padw push.BLK_HASH_PTR mem_loadw
 end
 
+#! Returns the block number of the reference block.
+#!
+#! Inputs:  []
+#! Outputs: [block_num]
+#!
+#! Where:
+#! - block_num is the number of the reference block for the transaction execution.
+export.get_block_num
```

Comment:
This needs to be tested somehow. LMK best way to do this

Reply:
To test the memory layout generally, you can extend the `global_input_memory_assertions` function in `crates/miden-tx/src/tests/kernel_tests/test_prologue.rs`.

This procedure specifically will be tested implicitly by testing procedures that call it, so no extra test needed.